code/_onclick/observer.dm

```diff
@@ -9,6 +9,11 @@
 
 	// Things you might plausibly want to follow
 	if(ismovable(A))
+		// It's a mimic! Fetch associated (real) atom
+		if(istype(A, /atom/movable/openspace/mimic))
+			var/atom/movable/openspace/mimic/M = A
+			check_orbitable(M.associated_atom)
+			return
 		check_orbitable(A)
```

Comment:
Actually I'm not really a big fan of this, I think there should be some base proc on /atom that handles orbitability and then have an override for the specific type. This would let us have more complex orbiting behaviours in the future (stuff like being able to see peoples UIs when you orbit them like in CM or something like that).

Reply:
You got it